lizzy/api.py

```diff
@@ -68,7 +70,8 @@ def create_stack(new_stack: dict) -> dict:
     keep_stacks = new_stack['keep_stacks']  # type: int
     new_traffic = new_stack['new_traffic']  # type: int
     image_version = new_stack['image_version']  # type: str
-    application_version = new_stack.get('application_version')  # type Optional[str]
+    application_version = new_stack.get('application_version')  # type: Optional[str]
+    stack_version = new_stack.get('stack_version')  # type: Optional[str]
```

Comment:
Currently I'm not using any static type checker, but PyCharm does type checking (and provides better auto completion) if the typing information is available.